editor.planx.uk/src/@planx/components/MapAndLabel/Public/Context.tsx

```diff
@@ -17,12 +19,13 @@ import { PresentationalProps } from ".";
 
 interface MapAndLabelContextValue {
   schema: Schema;
+  features?: Feature[];
   activeIndex: number;
   editFeature: (index: number) => void;
   formik: FormikProps<SchemaUserData>;
   validateAndSubmitForm: () => void;
   isFeatureInvalid: (index: number) => boolean;
-  addFeature: () => void;
+  removeFeature: (index: number) => void;
```

Comment:
I did consider making this something like `removeActiveFeature()` which would not require an argument and just use `activeIndex` internally.

I think passing an argument is a little more flexible, and would allow us to call this from other places if the remove button moved from the active tab (e.g. to the tab list).